# License header:

#**********************************************************************
# Copyright 2020 Advanced Micro Devices, Inc
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#********************************************************************

--- a/BlenderPkg/build_linux.sh.orig	2021-10-11 08:25:46.000000000 -0700
+++ b/BlenderPkg/build_linux.sh	2021-11-09 08:40:22.317671321 -0800
@@ -15,8 +15,10 @@
 # limitations under the License.
 #********************************************************************
 
+# oteodoro: Changed for generic python3 call for systems without python3.7.
+
 pushd ..
 ./build.sh
 popd
 
-python3.7 create_zip_addon.py
+python3 create_zip_addon.py # oteodoro: python3.7 -> python3
